cypress/support/step-definitions/date-input-steps.js

```diff
@@ -65,8 +65,16 @@ When('I choose date tomorrow via DayPicker', () => {
   dayPickerDay(TOMORROW_CALENDAR).click();
 });
 
+When('I type specific date {string}', (day) => {
+  dateInput().clear().type(`${day}${VALIDATION_DATE}`);
+});
+
 Then('the date is set to today', () => {
   dayPickerDay(TODAY_CALENDAR).should('have.attr', 'aria-label').should('contains', TODAY_CALENDAR);
   dayPickerDay(TODAY_CALENDAR).should('have.attr', 'class').should('contains', `${DAY_PICKER_PREFIX}today`);
   dayPickerDay(TODAY_CALENDAR).should('have.attr', 'class').should('contains', `${DAY_PICKER_PREFIX}selected`);
 });
+
+And('I click onto specific day {string} via DayPicker for validation component', (specificDay) => {
+  dayPickerDay(`${specificDay}`).click();
```

Comment:
No need to user `${specificDay}` we can pass only the `specificDay` parameter by itselfs

Reply:
fixed